### PR TITLE
[RW-1056][Risk=no] Add error message on name conflict cohort builder

### PR DESCRIPTION
**Pull request by blrubenstein**
Replace this this template with your PR description.
Please remember to keep in mind the security levels outlined in [CONTRIBUTING.md](CONTRIBUTING.md) and to 
include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title

* **no**: None 
* **low**: Low chance of potential impact to, or exposure of patient data
* **moderate**: Moderate chance of potential impact to, or exposure of patient data
* **severe**: Severe chance of potential impact to, or exposure of patient data

Please also:

* Get thumbs from reviewer(s)
* Verify all tests go green, including CI tests

### CODE REVIEW

**Comment by blrubenstein** [comment]
@dolbeew this just went through considerable revamp after product and design review! When you get another chance, PTAL.

File: ui/src/app/cohort-search/overview/overview.component.html [hunk 1]
```diff
@@ -72,16 +72,23 @@ <h2>Save New Cohort</h2>
...
+        <input
+          name="name"
+          id="name-control"
+          class="control"
+          placeholder="Cohort Name"
+          formControlName="name"
+          type="text"
+          [class.invalid]="(name.dirty || name.touched) && cohortForm.invalid"
+          (input)="showConflictError = false"
+          [ngClass]="{alreadyTaken: showConflictError}"
```
Comment (dolbeew) [inline ui/src/app/cohort-search/overview/overview.component.html]: nit: Should we use class binding here to be consistent with the above class on line 83?
`[class.alreadyTaken]="showConflictError"`

Or maybe set them both using ngClass?
`[ngClass]="{invalid: (name.dirty || name.touched) && cohortForm.invalid, alreadyTaken: showConflictError}"`

File: ui/src/app/cohort-search/overview/overview.component.html [hunk 2]
```diff
@@ -72,16 +72,23 @@ <h2>Save New Cohort</h2>
...
-        name="name"
-        id="name-control"
-        class="control"
-        placeholder="Cohort Name"
-        formControlName="name"
-        type="text"
-        [class.invalid]="(name.dirty || name.touched) && cohortForm.invalid"
-        (input)="showConflictError = false"
-        autofocus>
+      <div class="name-area" style="position: relative;">
```
[the remark this answers is not in the document]
Comment (blrubenstein) [reply ui/src/app/cohort-search/overview/overview.component.html]: Will do.

File: ui/src/app/cohort-search/overview/overview.component.html (hunk 1 above)
Comment (blrubenstein) [reply ui/src/app/cohort-search/overview/overview.component.html]: Sounds good, will convert over.

**Pull request merged**